scripts/ci/ci_mypy.sh

```diff
@@ -25,7 +25,7 @@ MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 basic_sanity_checks
 
-force_python_3_6
+force_python_3_5
```

Comment:
I think, we should add a version information to setup.cfg
```
[mypy]
python_version = 3.5
```

Reply:
It's not enough. I tried it in exact way you proposed. and if you do it, you will end-up with plenty of stubs missing. When you specify python version you likely have to also specify where to get the stubs from. 

I had no time to investigate, so I  decided to take a faster route. From the https://mypy.readthedocs.io/en/latest/common_issues.html#python-version-and-system-platform-checks:

"By default, mypy will use your current version of Python and your current operating system as default values for sys.version_info and sys.platform."

Happy to approve PR with working version with mypy.ini :) however.